lib/extend/helper.js

```diff
@@ -19,6 +20,17 @@ class Helper {
 
     this.store[name] = fn;
   }
+
+  setProp(name, prop) {
+    if (!name) throw new TypeError('name is required');
+    if (!prop) throw new TypeError('prop is required');
```

Comment:
It shouldn't be TypeError as this is not type checking.

Reply:
L18 uses `TypeError` as well.

Reply:
oh well, `undefined` can be considered as a type.